docs/content/preview/explore/transactions/distributed-transactions-ysql.md

```diff
@@ -226,7 +234,7 @@ yugabyte=# CREATE TABLE example(k INT PRIMARY KEY);
 ERROR: cannot execute CREATE TABLE in a read-only transaction
 ```
 
-### `DEFERRABLE` transactions
+### DEFERRABLE transactions
 
 The `DEFERRABLE` transaction property in YSQL is similar to PostgreSQL in that has no effect unless the transaction is also `SERIALIZABLE` and `READ ONLY`.
```

Comment:
_The DEFERRABLE transaction property in YSQL is similar to PostgreSQL in that has no effect unless the transaction is also SERIALIZABLE and READ ONLY._    -- ... in that has no effect unless ...  -- doesn't sound right